app/controllers/application_controller.rb

```diff
@@ -84,4 +100,8 @@ def redirect_to_root
   def staff?
     logged_in? && current_user.staff?
   end
+
+  def true_user
+    @true_user ||= User.find(session[:user_id]) if session[:user_id]
```

Comment:
Using a `defined?` check on `@true_user` instead of `||=` would save some AR calls.